api/envoy/config/rbac/v3/rbac.proto

```diff
@@ -218,6 +218,12 @@ message Permission {
     // Please refer to :ref:`this FAQ entry <faq_how_to_setup_sni>` to learn to
     // setup SNI.
     type.matcher.v3.StringMatcher requested_server_name = 9;
+
+    // A CIDR block that describes the upstream IP. This field is only applicable if the request is routed
+    // by dynamic forward proxy. If this field is present and the request did not have dynamic
```

Comment:
Can you explain in the documentation here what the resolution semantics are? What happens if there is a change in resolution during processing? What is the v4/v6 story? etc.